.github/workflows/certora.yml

```diff
@@ -13,20 +13,35 @@ on:
   workflow_dispatch:
 
 jobs:
-  verify_stkBNB:
+  get-diff:
     runs-on: ubuntu-latest
-
+    # Map a step output to a job output
+    outputs:
+      output1: ${{ steps.certora.outputs.run }}
```

Comment:
can name the outputs in a better way as per the job they will be used for. So, name it `certora` instead of `output1`

Reply:
combined certora to ci, updated.